build.sh

```diff
@@ -32,33 +32,33 @@ Usage: sh $0 command [argument]
 
 command:
   download-core:           downloads core library (binary version)
-  clean [xcmode]:          clean up/remove all generated files
-  build [xcmode]:          builds iOS and OS X frameworks with release configuration
-  build-debug [xcmode]:    builds iOS and OS X frameworks with debug configuration
-  ios [xcmode]:            builds iOS framework with release configuration
-  ios-debug [xcmode]:      builds iOS framework with debug configuration
-  ios-dynamic [xcmode]:    builds two iOS 8 dynamic frameworks: one for devices and one for the simulator
-  osx [xcmode]:            builds OS X framework with release configuration
-  osx-debug [xcmode]:      builds OS X framework with debug configuration
-  test-ios [xcmode]:       tests iOS framework with release configuration
-  test-ios-devices:        tests iOS on all attached iOS devices with release configuration
-  test-ios-devices-debug:  tests iOS on all attached iOS devices with debug configuration
-  test-osx [xcmode]:       tests OSX framework with release configuration
-  test [xcmode]:           tests iOS and OS X frameworks with release configuration
-  test-debug [xcmode]:     tests iOS and OS X frameworks with debug configuration
-  test-all [xcmode]:       tests iOS and OS X frameworks with debug and release configurations
-  examples [xcmode]:       builds all examples in examples/ in release configuration
-  examples-debug [xcmode]: builds all examples in examples/ in debug configuration
-  browser [xcmode]:        builds the Realm Browser OSX app
-  test-browser [xcmode]:   tests the Realm Browser OSX app
-  verify [xcmode]:         cleans, removes docs/output/, then runs docs, test-all, examples & browser
+  clean:                   clean up/remove all generated files
+  build: [settings]        builds iOS and OS X frameworks
+  ios: [settings]          builds iOS frameworks
+  ios-dynamic: [settings]  builds iOS dynamic frameworks
+  ios-static: [settings]   builds a fat iOS static framework
+  osx: [settings]          builds OS X framework
+  test-ios: [settings]     tests iOS framework
+  test-ios-devices:        tests iOS on all attached iOS devices
```

Comment:
Shouldn't this accept `[settings]` as well? Minor point: `[settings]` should come before the colon (e.g. `build [settings]: ...`).